RELEASE_NOTES.md

```diff
@@ -1,6 +1,8 @@
 ### Version 2.6.7 (TBD)
 
 * Upgrade to Gradle 2.11.
+* Bug fix when copying single file from container and hostPath is set to directory for `DockerCopyFileFromContainer` - [Pull request 163](https://github.com/bmuschko/gradle-docker-plugin/pull/163)
+* Step reports are now printed to stdout by default for `DockerBuildImage` - [Pull request 145](https://github.com/bmuschko/gradle-docker-plugin/pull/145)
```

Comment:
Not all tasks output information though. And in this specific case the output is something we have to purposefully cull out. I agree in that we should do it to be consistent but only where it makes sense. I don't see any value say for outputting the logs from DockerStopContainer but do see the value for DockerBuildImage.  The former gives us nothing while the latter tells us what the container is actually building so to speak.